pkg/types/rpm/v0.0.1/entry.go

```diff
@@ -107,6 +110,23 @@ func (v V001Entry) IndexKeys() []string {
 		}
 	}
 
+	// if this is a certificate, add email addresses to result
+	p, _ := pem.Decode(key)
+	if p != nil {
+		cert, err := x509.ParseCertificate(p.Bytes)
+		if err == nil {
+			var emailAddressOID asn1.ObjectIdentifier = []int{1, 2, 840, 113549, 1, 9, 1}
```

Comment:
Could you make this a constant somewhere and link to the definition of it for reference?